### PR TITLE
[STUD-215]Implement Appium & Initial Test Login

### PR DESCRIPTION
**Pull request by vithura14**

### CODE REVIEW

**Review by car12lin12 — COMMENTED** [review]
Can you provide steps on how to run your tests on our own machines?

File: App.js [hunk 1]
```diff
@@ -114,7 +115,7 @@ export default class App extends React.Component {
   }
 
   handleLogin(loginState) {
-    this.setState({ isLoggedIn: loginState });
+    this.setState({ isLoggedIn: false });
```
Comment (car12lin12) [inline App.js]: this prevents you from moving to next screen after login

**Comment by vithura14** [comment]
> Can you provide steps on how to run your tests on our

The steps and the necessary packaging are pretty complicated to go through. However, you can see the tests I am running and their result by running the following command: 
`allure open`

Here is a snapshot of the latest test results:
image
image

**Comment by car12lin12** [comment]
for those who want to run allure, do: npm install allure-commandline

**Review by car12lin12 — COMMENTED** [review]
Awesome work Vithu!! :partying_face: :partying_face: can you remove the package-lock.json changes and then we can merge?

**Comment by vithura14** [comment]
> Awesome work Vithu!! 🥳 🥳 can you remove the package-lock.json changes and then we can merge?

Thank you Carlin! It is removed :)

**Review by car12lin12 — APPROVED** [review]

**Pull request merged**